### DISCUSSION TITLE
atomic mass of hydrogen

### DISCUSSION THREAD

**Discussion by saeedshahi** [deepmodeling/deepmd-kit#1256] [Q&A] [answered]
Hello all,

The atomic mass of hydrogen is 1 so why do you consider 2 for that in the LAMMPS input sample file in the water example?

Best,
Saeed

**Comment by njzjz** [comment, marked as answer]
It's deuterium.

**Reply by saeedshahi** [reply]
Sorry, but I'm still confused about water MD. You did a comparison between MD  water by LAMMPS and Gromacs with different settings but used same  NN trained potential and get equal RDF. LAMMPS: Hydrogen mass = 2 g/mol and temperature = 330 K // Gromacs: Hydrogen mass = 1 g/mol and temperature = 298 K. Is it a right way for comparison or I'm wrong?

Best,
Saeed

**Reply by AnguseZhang** [reply]
No. I think when comparing rdf for GROMACS, LAMMPS and OPENMM, all conditions are set the same. The mass of hydrogen is set to 1.008.  @Ericwang6  Please verify. 

LAMMPS's input is just an example. We increase the mass of hydrogen to increase the stability of simulation. The reason is similar to Hydrogen Mass Repartitioning.
